### PR TITLE
Extending EditProfileForm

### PR DESCRIPTION
**Pull request by HKotsev**

### CODE REVIEW

File: storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/forms/default/profile.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+			label="label.input.interests.profile"
+			mandatory="true"
+			max-length="100"
+			missing-error="error.message.required"
+			range-error="error.message.100orless"
+			type="string"/> 
+
+			  <include formid="states" name="states" missing-error="address.state.missing" parse-error="error.message.required" value-error="error.message.required"/>
+
+		<field formid="countryOfResidence" label="label.input.country" type="string" mandatory="true"
```
Comment (isergeev19) [inline storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/forms/default/profile.xml:69]: Please watch out for correct formatting

File: storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/account/editProfileForm.isml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            <div class="form-group
+                ${pdict.profileForm.customer.countryOfResidence.mandatory === true ? 'required' : ''}">
+                <label class="form-control-label" for="countryOfResidence">
+                    <isprint value="${pdict.profileForm.customer.countryOfResidence.label}" encoding="htmlcontent" />
+                </label>
+                <select class="form-control" id="countryOfResidence"
+                    <isprint value="${pdict.profileForm.customer.countryOfResidence.attributes}" encoding="off" />
+                    autocomplete="countryOfResidence">
+                    <isloop items=${pdict.profileForm.customer.countryOfResidence.options} var="countryOfResidence">
+                        <option id="${countryOfResidence.id}" value="${countryOfResidence.htmlValue}" <isif condition="${countryOfResidence.selected}">selected</isif>>${countryOfResidence.label}</option>
```
Comment (isergeev19) [inline storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/account/editProfileForm.isml]: Will this .selected work?

File: storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/forms/default/profile.xml (hunk 1 above)
Comment (HKotsev) [reply storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/forms/default/profile.xml:69]: i will turn off the formating

File: storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/account/editProfileForm.isml (hunk 2 above)
Comment (HKotsev) [reply storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/account/editProfileForm.isml]: this is the part i wanted to fix

Comment (HKotsev) [reply storefront-reference-architecture-master/cartridges/app_storefront_custom_cartridge/cartridge/templates/default/account/editProfileForm.isml]: fixed

**Review by isergeev19 — APPROVED** [review]

**Pull request merged**